### DISCUSSION TITLE
Move IRC channel from freenode to libera?

### DISCUSSION THREAD

**Discussion by solvingj** [pyinstaller/pyinstaller#7013] [PyInstaller] [answered]
Many users and projects have abandoned freenode recently over major security and safety concerns:

https://arstechnica.com/gadgets/2021/05/freenode-irc-has-been-taken-over-by-the-crown-prince-of-korea/
https://www.techradar.com/news/major-open-source-projects-abandon-freenode-following-hostile-takeover

Most have migrated to libera

https://en.wikipedia.org/wiki/Libera_Chat

This page suggests that this project still uses freenode, although I did not go to freenode to see if it's still active:

https://github.com/pyinstaller/pyinstaller/wiki/Community

Anyway, if this project still intends to use IRC, this is a suggestion to make an official switch.

**Comment by bwoodsend** [comment]
Uhm, this is the first I've heard that we even have an IRC. @Legorooj Do you know anything about this?

**Reply by Legorooj** [reply]
Yes, technically we have an IRC. However I removed all mention of this in our docs a while back as the IRC channel is not used and hasn't been since way before I started working on PyInstaller.

**Reply by Legorooj** [reply]
Appears I missed the wiki though.

**Comment by Legorooj** [comment, marked as answer]
@solvingj thanks for bringing this up, however as we no longer officially support our IRC channel (github issues/discussions and the mailing list more than suffice) we won't be changing anything. I've removed all mentions of IRC from the wiki.

**Reply by bwoodsend** [reply]
It looks like the channel we did have has been deleted anyway. I can't get onto it. So all ok :slightly_smiling_face:
